### DISCUSSION TITLE
How to run PHPCSFixer on PHP 8.2?

### DISCUSSION THREAD

**Discussion by dev-bajpai** [PHP-CS-Fixer/PHP-CS-Fixer#7192] [Q&A] [answered]
PHP needs to be a minimum version of PHP 7.4.0 and maximum version of PHP 8.1.*.
Current PHP version: 8.2.8.
how to resolve this type of problem

**Comment by Wirone** [comment, marked as answer]
Hi @dev-bajpai, you need to update Fixer to at least [v3.15](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/releases/tag/v3.15.0) in which support for PHP 8.2 was added. Otherwise you need to set `PHP_CS_FIXER_IGNORE_ENV=1`.
